pyartifactory/objects/artifact.py

```diff
@@ -75,7 +76,12 @@ def info(self, artifact_path: Union[Path, str]) -> ArtifactInfoResponse:
                 raise ArtifactNotFoundError(f"Artifact {artifact_path} does not exist")
             raise ArtifactoryError from error
 
-    def deploy(self, local_file_location: Union[Path, str], artifact_path: Union[Path, str]) -> ArtifactInfoResponse:
+    def deploy(
+        self,
+        local_file_location: Union[Path, str],
+        artifact_path: Union[Path, str],
+        checksums: bool = False,
```

Comment:
Just `checksum` in singular to comply with Artifactory naming **Deploy Artifact by Checksum**

Reply:
Or even better, `checksum_enabled: bool = False`